external/radiation/python/radiation_astronomy.py

```diff
@@ -50,7 +49,7 @@ def __init__(self, me, isolar, solar_filename):
                     " with cycle approximation (old values)!",
                 )
 
-            file_exist = os.path.isfile(os.path.join(FORCING_DIR, self.solar_fname))
```

Comment:
So is this needed because the `solar_fname` being provided already has the forcing dir now?

Reply:
FORCING_DIR is imported from config so there were conflicts when I used a different path.